### ISSUE TITLE
Setting limitInputPixels for composite input images

### ISSUE THREAD

**Issue by mariosvlad** [lovell/sharp#2099]
Hi, not sure if this is a bug or it was an unintended feature that stopped working, in v0.23.3 this worked when the composite input image (image2Path) had size larger than the default limit (16k by 16k):
```
await sharp(image1Path)
    .limitInputPixels(false)
    .composite([
        {
            input: image2Path,
            ...
        },
    ])
    .toFile(outputPath);
```
But on v0.24.1 it throws `Input image exceeds pixel limit` error.
Also the `limitInputPixels` function is deprecated so there is no way to specify it for the composite input images, one possible solution is by accepting sharp object #2025, or maybe by adding `limitInputPixels` in composite parameters?

**Comment by lovell** [comment]
Sorry about this, `limitInputPixels` (and `sequentialRead`) were supposed to have been exposed on to the `composite` operation.

This will be fixed in the forthcoming v0.25.0, which will allow you to use the following:

```javascript
    .composite([
        {
            input: image2Path,
            limitInputPixels: false
            ...
        },
    ])
```

**Comment by lovell** [comment]
v0.25.0 now available, thank you for (re-)reporting this.
